packages/account/src/Components/trading-assessment/trading-assessment-form.tsx

```diff
@@ -17,16 +29,20 @@ const TradingAssessmentForm = ({
     should_move_to_next,
     setSubSectionIndex,
     is_independent_section,
-}) => {
+}: TTradingAssessmentForm) => {
     const [is_section_filled, setIsSectionFilled] = React.useState(false);
     const [should_inform_user, shouldInformUser] = React.useState(false);
+    const [form_data, setFormData] = React.useState({ ...form_value });
+
+    const assessment_questions = trading_assessment_questions();
+
+    const stored_items = parseInt(localStorage.getItem('current_question_index') || '0');
+
```

Comment:
stored_items is a bit ambiguous, can we rename to storage key `current_question_index`?